### ISSUE TITLE
Jaw hyperreflexia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6722]
**Preferred term label:**
Jaw hyperreflexia

**Synonyms**
Brisk jaw jerk

**Definition (free text, please give PubMed ID)**
Increased intensity of muscle tendon reflexes in jaw.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:128230 
PMID:16908750  2/4

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
